.github/workflows/mac.yml

```diff
@@ -70,10 +70,10 @@ jobs:
           fetch-depth: 2
       - name: Run sccache-cache
         uses: mozilla-actions/sccache-action@v0.0.3
-      - name: Select Python 3.11
+      - name: Select Python 3.12
         uses: actions/setup-python@v4
         with:
-          python-version: '3.11'
+          python-version: '3.12'
```

Comment:
We might want to remove this code altogether before landing.

Reply:
Okay. I've removed this in the latest version of the branch. It's still very hard to ensure a Python 3.12 runner when testing.